From c0d1dba5a995e80a68a55db50196d7751bd4e09c Mon Sep 17 00:00:00 2001
From: Zhangbin Tong <zebulun.tong@rock-chips.com>
Date: Tue, 18 Sep 2018 08:35:14 +0800
Subject: [PATCH] arm64: rockchip_defconfig: Enable CONFIG_COREDUMP

vts-testcase/kernel/api/proc/KernelProcFileApiTest
had added tests for the core_pattern.

https://android.googlesource.com/kernel/configs
5ed73dc4d482 ("add missing configs")

CONFIG_COREDUMP and CONFIG_ELF_CORE are recommended for diagnostic
purposes.

came from android-recommended.config

Change-Id: I8400ffff1b5e025b4967a39536082fbcc4ec0859
Signed-off-by: Zhangbin Tong <zebulun.tong@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index a01ad8a7b8f0..93247f04e20e 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -73,7 +73,7 @@ CONFIG_CP15_BARRIER_EMULATION=y
 CONFIG_SETEND_EMULATION=y
 CONFIG_ARM64_SW_TTBR0_PAN=y
 # CONFIG_EFI is not set
-# CONFIG_COREDUMP is not set
+# CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS is not set
 CONFIG_COMPAT=y
 CONFIG_PM_AUTOSLEEP=y
 CONFIG_PM_WAKELOCKS=y
-- 
2.35.3

